buildpipeline/pipelines.json

```diff
@@ -354,6 +354,39 @@
             "PB_BuildType": "Release"
           }
         },
+		{
+          "Name": "Dotnet-CoreClr-Trusted-BuildTests",
```

Comment:
Instead of adding this new block, you should just add "Windows.10.amd64.Core" to the "TargetQueues" list here - https://github.com/dotnet/coreclr/blob/012a404300e459a5d8115beba6f2c843950bbaca/buildpipeline/pipelines.json#L330 by using a comma separator (See the Linux jobs below for an example)